openmc/settings.py

```diff
@@ -139,6 +139,8 @@ class Settings(object):
         The elastic scattering model to use for resonant isotopes
     volume_calculations : VolumeCalculation or iterable of VolumeCalculation
         Stochastic volume calculation specifications
+    no_nu : bool
+        Indicate do not creat fission sites.
```

Comment:
Consider rewording: "Indicate whether fission neutrons should be created or not."